index.html

```diff
@@ -127,8 +127,8 @@
 Spec developers, authors, and reviewers can use it to guide their thinking.
 In particular, the purpose of this document is to
 inform wide review of new charters,
-new specifications, and
-updates to published recommendations.
+new technical and community reports, and
+updates to published reports.
```

Comment:
I think we ought to keep mentioning specifications and recommendations, since non-W3C readers might not realize that "reports" is a superset (they might also not realize that "standards" are included in "recommendations"), but +1 on broadening the purpose like this.

Perhaps:

> the purpose of this document is to inform wide review of
> charters and reports produced by the web community,
> including specifications, W3C recommendations, and technical and non-technical reports.

(Apologies that Github doesn't let me make this a suggested change.)

The categories in that last line overlap, and I'm wondering if it'll be more confusing to list them as-is, or to use extra words to point out that they overlap.